modules/support/src/lib.rs

```diff
@@ -327,7 +327,13 @@ pub trait PrecompileCallerFilter {
 
 /// An abstraction of EVM for EVMBridge
 pub trait EVM<AccountId> {
-	type Balance: AtLeast32BitUnsigned + Copy + MaybeSerializeDeserialize + Default;
+	type Balance: AtLeast32BitUnsigned
+		+ TryFrom<u32>
+		+ TryFrom<u64>
+		+ TryFrom<u128>
```

Comment:
`AtLeast32BitUnsigned` should already include those

Reply:
@xlc yes you're right